matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/message/MessageLocationContent.kt

```diff
@@ -40,72 +40,45 @@ data class MessageLocationContent(
         @Json(name = "geo_uri") val geoUri: String,
 
         @Json(name = "m.relates_to") override val relatesTo: RelationDefaultContent? = null,
-        @Json(name = "m.new_content") override val newContent: Content? = null
+        @Json(name = "m.new_content") override val newContent: Content? = null,
+        /**
+         * See https://github.com/matrix-org/matrix-doc/blob/matthew/location/proposals/3488-location.md
+         */
+        @Json(name = "org.matrix.msc3488.location") val unstableLocationInfo: LocationInfo? = null,
+        @Json(name = "m.location") val stableLocationInfo: LocationInfo? = null,
```

Comment:
Please only use `unstable` prefix, and never use `stable` prefix.
Applicable to all the fileds in this PR.

Reply:
We should be able to read the stable one too. When MSC is finalized and we start supporting stable prefixes then older versions should be able to read these fields. Do I miss something here?

Reply:
Yes, sorry, I was not clear.
What I want to see here is:
```kotlin
        @Json(name = "org.matrix.msc3488.location") val unstableLocationInfo: LocationInfo? = null,
        @Json(name = "m.location") val locationInfo: LocationInfo? = null,
```

`stableLocationInfo` -> `locationInfo`

Reply:
Then we need an alternative naming for `fun getLocationInfo()`

Reply:
What do you think about `getBestLocationInfo()`

Reply:
That's OK.
Ideally those management should be quite hidden in the SDK. But it's fine like this I guess.